chunks/tool-yugabytedb/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN curl -sSLo ./yugabyte.tar.gz https://downloads.yugabyte.com/releases/${YB_VE
 ENV PATH="$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH"
 ENV PYENV_ROOT="$HOME/.pyenv"
 RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv \
-	&& git -C ~/.pyenv checkout ff93c58babd813066bf2d64d004a5cee33c0f27b \
+	&& git -C ~/.pyenv checkout ff93c58babd813066bf2d64d004a5cee33c0f27b # v2.4.10 \
```

Comment:
ff93c58babd813066bf2d64d004a5cee33c0f27b is https://github.com/pyenv/pyenv/commits/v2.3.10/

Forgot to update commit sha?

Reply:
Yes! 🤦‍♂️
I've been playing with rebases too much haha